.github/workflows/zap-scan.yml

```diff
@@ -0,0 +1,34 @@
+name: Zap Scan
+
+on:
+  push:
```

Comment:
If we end up merging the dev pipeline action, I can update this to run after the pipeline, and so for the first time we'll actually be running the zap scan against the deployed code instead of the code before the site is deployed (current state)